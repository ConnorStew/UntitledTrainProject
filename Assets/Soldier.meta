fileFormatVersion: 2
guid: ed31bd4e2f9f8284eb0a48c47509fa69
folderAsset: yes
DefaultImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
