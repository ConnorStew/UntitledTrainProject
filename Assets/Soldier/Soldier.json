{
  "conversations": [
    {
      "dialog": "Hey there, good to see a fresh face around here. I'm Andy hope ya get settled in. I'm always ‘round if ya feel the need for a drink or a hand of cards. I got some fresh bread here too, feel free to take some if ya get hungry."
    },

    {
      "dialog": "Well hope ya get settled in. I’m always around if ya feel the need for a drink of a hand of cards, got some fresh bread here, feel free to take some if ya get hungry.",
      "links": [
        {
          "word": "bread",
          "goto": "Bread"
        }
      ]
    },

    {
      "choices": [
        {
          "choice": "How long have you been here?",
          "response": "Just over a couple of weeks. It's a nice enough ride, though I’m not sure where we're headed, and that driver kinda gives me the willies.",
          "links": [
            {
              "word": "driver",
              "goto": "Driver"
            },
            {
              "word": "willies",
              "goto": "Willies"
            }
          ]
        },
        {
          "choice": "Have you met any of the other passengers?",
          "response": "Yeah that Hunter aint got much class, but she's a hoot and a half I'll tell ya that. Got stories that'll make any man squirm like a racoon in the wash. The lil’ fellas a tad odd for my taste seems decent enough but i don't trust him much. Always just sorta staring at ya, like he's seein right through ya.",
          "links": [
            {
              "word": "Hunter",
              "goto": "Hunter"
            },
            {
              "word": "stories",
              "goto": "Stories"
            },
            {
              "word": "staring",
              "goto": "Stare"
            }
          ]
        },
        {
          "choice": "Do you know anything about this train?",
          "response": "Not much, no. Driver told me this was a place of change and possibility but kept saying I should just enjoy the ride. Had a good look ‘round of course, It's freaky though carriages just seem to appear when you want them.",
          "links": [
            {
              "word": "appear",
              "goto": "Appear"
            }
          ]
        },
        {
          "choice": "What did you do before this?",
          "response": "Well… been a soldier most my life. Got made a sergeant after my C.O was taken out. Never expected when I signed up that I'd be in charge of anyone.",
          "links": [
            {
              "word": "out",
              "goto": "Out"
            },
            {
              "word": "expected",
              "goto": "Expected"
            },
            {
              "word": "charge",
              "goto": "Charge"
            }
          ]
        }
      ]
    }
  ]
}