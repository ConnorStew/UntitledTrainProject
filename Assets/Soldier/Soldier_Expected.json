{
  "conversations": [
    {
      "dialog": "Well, what were you expecting?",
      "character": "Player"
    },

    {
      "dialog": "Just to do my part, come home, try something else maybe. But it just kept getting delayed. By the time Melissa got taken, we were so deep in it. We knew we weren't goin home.",
      "links": [
        {
          "word": "somthing",
          "goto": "Somthing"
        },
        {
          "word": "Melissa",
          "goto": "Melissa"
        }
      ]
    }
  ]
}